content/References/ZFSPrimer.md

```diff
@@ -1,179 +1,216 @@
+As a general rule, do not add L2ARC to a system with less than 32 GiB of RAM, and the size of an L2ARC should not exceed ten times the amount of RAM. 
+In some cases, it might be more efficient to have two separate pools: one on SSDs for active data, and another on hard drives for rarely used content. 
 After adding an L2ARC device, monitor its effectiveness using tools such as `arcstat`.
-To increase the size of an existing L2ARC, stripe another cache device with it.
-The web interface will always stripe L2ARC, not mirror it, as the contents of L2ARC are recreated at boot.
-Failure of an individual SSD from an L2ARC pool will not affect the integrity of the pool, but may have an impact on read performance, depending on the workload and the ratio of dataset size to cache size.
-Note that dedicated L2ARC devices cannot be shared between ZFS pools.
+To increase the size of an existing L2ARC, stripe another cache device with it. 
+The web interface always stripes L2ARC, it does not mirror it, as the contents of L2ARC are recreated at boot. 
+Failure of an individual SSD from an L2ARC pool does not affect the integrity of the pool, but might have an impact on read performance, depending on the workload and the ratio of dataset size to cache size. 
+Note you cannot share dedicated L2ARC devices between ZFS pools.
 
-**ZFS was designed to provide redundancy while addressing some of the inherent limitations of hardware RAID** such as the write-hole and corrupt data written over time before the hardware controller provides an alert.
-ZFS provides three levels of redundancy, known as *RAIDZ*, where the number after the *RAIDZ* indicates how many disks per vdev can be lost without losing data.
-ZFS also supports mirrors, with no restrictions on the number of disks in the mirror.
-ZFS was designed for commodity disks so no RAID controller is needed.
-While ZFS can also be used with a RAID controller, it is recommended that the controller be put into JBOD mode so that ZFS has full control of the disks.
+### ZFS Redundancy and RAID
 
-When determining the type of ZFS redundancy to use, consider whether the goal is to maximize disk space or performance:
+ZFS was designed to provide redundancy while addressing some of the inherent limitations of hardware RAID such as the write-hole and corrupt data written over time before the hardware controller provides an alert.
+ZFS provides three levels of redundancy, known as RAIDZ, where the number after the RAIDZ indicates how many disks per vdev you can lose without losing data. 
+ZFS also supports mirrors, with no restrictions on the number of disks in the mirror. 
+ZFS was designed for commodity disks so no RAID controller is needed. 
+While you can also use ZFS with a RAID controller, it is recommended that you put the controller into JBOD mode so that ZFS has full control of the disks. 
```

Comment:
```suggestion
ZFS provides redundancy while addressing some inherent limitations of hardware RAID, such as the write-hole and corrupt data written over time before the hardware controller provides an alert.
ZFS provides three levels of redundancy, known as RAIDZ, where the number after the RAIDZ indicates how many disks per vdev you can lose without losing data. 
ZFS also supports mirrors, with no restrictions on the number of disks in the mirror. 
ZFS uses commodity disks and does not need a RAID controller. 
While you can also use ZFS with a RAID controller, we recommend that you put the controller into JBOD mode so that ZFS has full disk control. 
```